becca-new-app/src/App.js

```diff
@@ -22,11 +23,14 @@ async function fetchData(setTodoList, setIsLoading) {
 		}
 
 		const result = await response.json();
+		console.log("Airtable API Response:", result);
 		const fetchedTodoList = result.records || [];
+		console.log("Fetched Todo List from Airtable:", fetchedTodoList);
 
-		const todos = fetchedTodoList.map((todo) => ({
-			title: todo.fields.title,
-			id: todo.id,
+		const todos = fetchedTodoList.map((record) => ({
+			title:
+				record.fields && record.fields.Title ? record.fields.Title : "Untitled",
```

Comment:
@PixelBayBuilds - the reason why I was getting "Untitled" on mine was because of this line.

Why is it `.Title` with a capital T? Please revert the field back to lowercase T so it would work for both of us.

Reply:
Also, as a reminder, please check https://github.com/PixelBayBuilds/react-todo/pull/11 as well since I left comments a while back that are unresolved

Reply:
Thank you I made the changes here

https://github.com/PixelBayBuilds/react-todo/pull/11/commits/8390efef0297cd04cf1bd6788db0aec4555f25f6